scripts/SREP/get-sosreport/script.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+## validate input
+if [[ -z "$NODE" ]]
+then
+    echo "Variable node cannot be blank"
+    exit 1
+fi
+
+# Import sftp_upload library
+source /managed-scripts/lib/sftp_upload/lib.sh
+
+# Define expected values
+DUMP_DIR="${PWD}"
+DATE="$(date -u +"%Y%m%dT%H%M")"
+SOSREPORT_FILENAME="sosreport-${NODE}.tar.xz"
+SOSREPORT_FILEPATH="${DUMP_DIR}/sosreport-${NODE}.tar.xz"
+
+
+
+check_node(){
+    echo "Checking if \"${NODE}\" is an existing node..."
+    
+    if (oc get nodes -l node-role.kubernetes.io/worker= -oname | grep "${NODE}") &> /dev/null; then
```

Comment:
wonder if that must be a worker node to collect sosreport?

Reply:
Agree, I will adjust the check to validate if is an existing node rather against the worker label.